src/main/java/service/ReportServiceImp.java

```diff
@@ -0,0 +1,37 @@
+package service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ReportServiceImp implements ReportService {
+    private static final String MINUS_ACTIVITIES_CODES = "p";
+
+    @Override
+    public List<String> makeBalanceReport(List<String> activitiesList) {
+        int sum;
+        List<String> result = new ArrayList<>();
+        result.add("fruit,quantity");
+
+        Map<String, List<String[]>> stringListMap = activitiesList.stream()
+                .map(line -> line.split(","))

```

Comment:
move ',' to constant SEPARATOR - this way we can reuse it in different classes and it will improve readability